src/main/java/dan200/computercraft/core/computer/ComputerExecutor.java

```diff
@@ -174,6 +174,7 @@
         apis.add( new FSAPI( environment ) );
         apis.add( new PeripheralAPI( environment ) );
         apis.add( new OSAPI( environment ) );
+        apis.add( new JSONAPI() );
```

Comment:
Is it possible to have this use require instead of APIs? I can see this conflicting with some older programs which test for and expect a different JSON API.

Reply:
Not for Java ones. Honestly, my inclination has always been to define this in Lua - it's much more of a pain, but I like to keep the Java side of things as minimal as possible.

Reply:
I think thats one of the rare cases that could be done better on Java side.  As I already said, we have already a lib for that in java. In Lua we have to write everything from scratch.Most of the JSON APIs for CC out there have bugs and did not work with every file. With gson, we can be sure that  everything works and we only need a few lines of code.

Reply:
As far as I'm aware, the main with existing JSON APIs is that they don't handle all string escapes correctly, especially UTF-8 escapes. Once #363 is closed, that should be much easier to get right.

I'll have a fiddle over the weekend and see - I have a big deadline coming up this Friday, so a little preoccupied with that :).